Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.14-alpine3.12 as manifest-tool-builder
```

Comment:
if you DO need manifest-tool, you can pull the binary from the releases and not use `go get`

https://github.com/estesp/manifest-tool/releases